src/App.js

```diff
@@ -1,12 +1,37 @@
 import React from 'react';
+import {
+  BrowserRouter as Router, Route, Routes, Link,
+} from 'react-router-dom';
+import Home from './components/Home';
 import Calculator from './components/Calculator';
+import DadJoke from './components/DadJoke';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <Calculator />
-    </div>
+    <Router>
+      <nav>
+        <span className="page-name">
+          Math Magicians
+        </span>
+        <ul>
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="/calculator">Calculator</Link>
+          </li>
+          <li>
+            <Link to="/quote">Quote</Link>
+          </li>
+        </ul>
+      </nav>
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/calculator" element={<Calculator />} />
```

Comment:
- [ ] Well done on fulfilling practically all of the project's requirements 👏. Your `Calculator` page, however, differs slightly from the wireframe. We can see a text to the left of the app on the wireframe. Can you kindly consider doing the same thing in order to match the wireframe as requested? 🙏

### Wireframe :point_down: 
![Screenshot from 2023-09-28 11-47-15](https://github.com/Pi1998/math-magic-react/assets/89228743/be286d79-fbb5-40de-af2f-0cc4d7a9f8a9)

### your page :point_down: 
![Screenshot from 2023-09-28 12-10-17](https://github.com/Pi1998/math-magic-react/assets/89228743/a195c5ee-658f-4377-b50c-3e0026422273)